### PR TITLE
add registration scenario

### PR DESCRIPTION
**Pull request by AnLozovskaya**

### CODE REVIEW

File: TESTS/store_test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Feature('store');
+
+Scenario ('test eng',  ({ I, homePagePage, registerPage }) => {
+    I.openStore();
+    homePagePage.selectWebsiteEngLang();
+    homePagePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registratedUser.email = Date.now() + '@test.com';
+    registerPage.fillRegistrationDetails(registratedUser);
+    pause();
```
Comment (adudyak) [inline TESTS/store_test.js]: remove `pause()`

File: TESTS/store_test.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+let registratedUser = {
+    firstName: 'Ann',
+    lastName: 9,
+    email: 'qatest1287@gmail.com',
+    telephone: 4372552055,
+    password: 'Testing987654!',
+}
```
Comment (adudyak) [inline TESTS/store_test.js]: `;` is missing

File: TESTS/store_test.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Scenario ('test eng',  ({ I, homePagePage, registerPage }) => {
+    I.openStore();
+    homePagePage.selectWebsiteEngLang();
+    homePagePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registratedUser.email = Date.now() + '@test.com';
+    registerPage.fillRegistrationDetails(registratedUser);
+    pause();
+    });
```
Comment (adudyak) [inline TESTS/store_test.js:17]: rename folder to lower case `tests`

File: TESTS/store_test.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+}
+Feature('store');
+
+Scenario ('test eng',  ({ I, homePagePage, registerPage }) => {
+    I.openStore();
+    homePagePage.selectWebsiteEngLang();
+    homePagePage.openRegistrationPage();
+    registerPage.verifyRegisterAccountText();
+    registratedUser.email = Date.now() + '@test.com';
+    registerPage.fillRegistrationDetails(registratedUser);
```
Comment (adudyak) [inline TESTS/store_test.js]: rename function to `registerUser()` since it is not only fills details

File: TESTS/store_test.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+let registratedUser = {
+    firstName: 'Ann',
+    lastName: 9,
+    email: 'qatest1287@gmail.com',
+    telephone: 4372552055,
+    password: 'Testing987654!',
+}
+Feature('store');
+
+Scenario ('test eng',  ({ I, homePagePage, registerPage }) => {
```
Comment (adudyak) [inline TESTS/store_test.js]: name scenario `register new user`

File: codecept.conf.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+// turn on headless mode when running with HEADLESS=true environment variable
+// export HEADLESS=true && npx codeceptjs run
+setHeadlessWhen(process.env.HEADLESS);
+
+// enable all common plugins https://github.com/codeceptjs/configure#setcommonplugins
+setCommonPlugins();
+
+/** @type {CodeceptJS.MainConfig} */
+exports.config = {
+  tests: './TESTS/*_test.js',
```
Comment (adudyak) [inline codecept.conf.js]: here folder needs to be updated as well

File: pages/register.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+  lastNamefield: { xpath: '//*[@id="account"]/div[3]/label' },
+  emailField: { xpath: '//*[@id="account"]/div[4]/label' },
+  telephoneField: { xpath: '//*[@id="account"]/div[5]/label' },
+  passwordField: { xpath: '//*[@id="content"]/form/fieldset[2]/div[1]/label' },
+  termsRadiobutton: { xpath: '//*[@id="content"]/form/div/div/input[1]' },
+  passConfirmField: { xpath: '//*[@id="content"]/form/fieldset[2]/div[2]/label' },
+  continueButton: { xpath: '//*[@id="content"]/form/div/div/input[2]' },
+  registrationconfirmationTextEng: 'Your Account Has Been Created!',
+
+  verifyRegisterAccountText() {
```
Comment (adudyak) [inline pages/register.js:15]: function can be removed

**Review by adudyak — APPROVED** [review]

File: TESTS/store_test.js (hunk 5 above)
Comment (AnLozovskaya) [reply TESTS/store_test.js]: Done

File: codecept.conf.js (hunk 6 above)
Comment (AnLozovskaya) [reply codecept.conf.js]: Done

File: pages/register.js (hunk 7 above)
Comment (AnLozovskaya) [reply pages/register.js:15]: tbd

File: TESTS/store_test.js (hunk 3 above)
Comment (AnLozovskaya) [reply TESTS/store_test.js:17]: Done

File: TESTS/store_test.js (hunk 2 above)
Comment (AnLozovskaya) [reply TESTS/store_test.js]: Done

File: TESTS/store_test.js (hunk 1 above)
Comment (AnLozovskaya) [reply TESTS/store_test.js]: Done

**Review by adudyak — APPROVED** [review]

**Pull request merged**